README.md

```diff
@@ -101,10 +101,10 @@ And lastly, prebuild the application so the indy-sdk can be added as native depe
 
 ```sh
 # yarn
-yarn prebuild
+yarn expo prebuild
 
 # npm
-npm run prebuild
+./node_modules/.bin/expo prebuild
```

Comment:
Their docs just mention using `npx expo prebuild` for both package managers. 

Doing `yarn expo prebuild` is fine I think, but maybe we can update the npm one use npx:

```suggestion
npx expo prebuild
```